### PR TITLE
Release notes 2.2

### PR DESCRIPTION
**Pull request by joakim-hove**
**Task**
Add release notes for the 2.2 release.

**Approach**
Have updated the download-pr script slightly, and then manually edited the resulting text.

### CODE REVIEW

File: release-notes.txt [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+ - Added NUMRES values to EGRID header [125].
+
+Build & testing:
+
+ - Removed warnings - added pylint [4, 5, 6, 10, 11, 12]
+ - Accept any Python 2.7.x version [17, 18]
+ - Remove ERT testing & building [3, 19]
+ - Changes to Python/cmake machinery [25, 30, 31, 32, 49, 52, 62].
+ - Added cmake config file [33, 44, 45, 47].
+ . Only *one* library [54, 55, 58, 69, 73, 77, 91, 133]
```
Comment (markusdregi) [inline release-notes.txt]: `.` -> `-`

File: release-notes.txt [hunk 2]
```diff
@@ -1,5 +1,70 @@
...
+ - Rmoved stale binaries [59].
```
Comment (markusdregi) [inline release-notes.txt]: Removed

File: release-notes.txt [hunk 3]
```diff
@@ -1,5 +1,70 @@
...
+ - Use -std=c++0x if -std=c++11 is unavailable [118]
+ - Make all of (previous( libutil compile with C++ [162]
+
+Well:
+
+ - Get well rates from restart files [8,20].
+ - Test if file exists before load [111].
+ - Fix some warnings [169]
+
+Core:
```
Comment (markusdregi) [inline release-notes.txt]: Should we mention that we also now support MacOS, as all tests are passing? In addition the library compiles with `-werror=all` on Travis, meaning that we have fewer warnings.

**Comment by joakim-hove** [comment]
Small comments adressed - merging.

**Pull request merged**